test/juttled/juttled.spec.js

```diff
@@ -85,7 +85,8 @@ describe("Juttled Tests", function() {
             return start_job_check_job_id();
         });
 
-        it("Start 2 jobs at once, should return both jobs", function() {
+        it("Start 2 jobs at once, should return both jobs", function(done) {
+            this.timeout(5000);
```

Comment:
Horrifying! I set it to 30s in https://github.com/juttle/outrigger/pull/73 though, so think we should stick with that.